src/s3ql/backends/b2/b2_error.py

```diff
@@ -19,14 +18,7 @@ def __init__(self, status, code, message, headers=None):
             self.retry_after = 1
```

Comment:
no need for `_parse_retry_after_header` here, just change it to
```python
        # Force 1s waiting time before retry
        if not self.retry_after:
            self.retry_after = 1
```
since `super().__init__()` did that already.